src/components/ebay-breadcrumb/README.md

```diff
@@ -20,7 +20,7 @@ Name | Type | Stateful | Description
 
 Event | Description | Data
 --- | ---
-`breadcrumb-click` | click breadcrumb items | `{ event, target:event.target, currentTarget: el }`
+`breadcrumb-select` | click breadcrumb items | `{ event, target:event.target, currentTarget: el }`
```

Comment:
For consistency, I think we should make the event signature match `menu-select`:

`emitAndFire(this, 'menu-select', { index: itemIndex, checked: [itemIndex], el: itemEl });`

(replace `checked` with `current`)

Pagination would also be similar:

`emitAndFire(this, 'pagination-select', { index: itemIndex, current: [itemIndex], el: itemEl });`

In next release we can discuss how to make it easier to prevent the default click action.

Reply:
can we just pass `el` alone like below. i think passing index does not make sense for links on breadcrumb. thought?

```emitAndFire(this, 'breadcrumb-select', {el: event.target});```

Reply:
done as we discussed `emitAndFire(this, 'breadcrumb-select', {el: event.target});`

Reply:
okay, I guess we can add the index and currentItem in future if necessary.